KULeuvenNeuromechanics/PredSim
# New heelstrike detection

Implemented new algorithm to detect heelstrike and reconstruct the gait cycle.
* method is more tractable
* prevents errors where heelstrike was not found for some simulations
* solves bugs with full gait cycle simulations  closes #72  

How to test
1) run [unit test](https://github.com/KULeuvenNeuromechanics/PredSim/blob/new_heelstrike_detection/Tests/test_getStancePhaseSimulation.m)
2) redo heelstrike detection for existing simulation results (half and full gait cycle)
`S.post_process.load_prev_opti_vars = 1;`
`% S.post_process.rerun   = 1;` should be commented or set to 0
`S.post_process.result_filename = 'Falisse_et_al_2022_v9';` your result that you want to redo
